tests/test_core.py

```diff
@@ -23,9 +23,19 @@
 from molecule import core
 
 
+class TestProvisioner():
+    def __init__(self):
+        self.instances = [{'name': 'demo-01',
+                           'ansible_groups': ['demo', 'demo1'],
+                           'options': {'append_platform_to_hostname': True}}]
+        self._platform = 'centos7'
```

Comment:
Oh, yeah good point.  There are probably ways to do it, but I'm not going to worry about it.